packages/bindings-test/src/posts/mod.rs

```diff
@@ -1,2 +1,68 @@
 mod msg;
 mod query;
+
+use crate::chain_communication::DesmosCli;
+use cosmwasm_std::Addr;
+
+use test_contract::msg::QueryMsg::DesmosChain;
+
+use desmos_bindings::cosmos_types::PageRequest;
+use desmos_bindings::posts::msg::PostsMsg;
+use desmos_bindings::posts::types::{
+    Entities, Post, QuerySubspacePostsRequest, QuerySubspacePostsResponse, ReplySetting, Url,
+};
+
+/// Creates a post inside the given subspace for interaction testing
+pub fn create_test_post(subspace_id: u64, contract_address: &str) -> Post {
```

Comment:
To make test stateless, we can create a post then return it for the test, then we can avoid from a lot of const number like TEST_MOVABLE_POST and etc.